index.html

```diff
@@ -1,41 +1,217 @@
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul>
+              <li class="list-sumbua">
+                CSS <span> <img src="images/image3.png" alt="css" /></span>
+              </li>
+              <li class="list-sumbua">
+                Html <span> <img src="images/image3.png" alt="html" /></span>
+              </li>
+              <li class="list-sumbua">
+                Bootstrap
+                <span> <img src="images/image3.png" alt="bootstrap" /></span>
+              </li>
+              <li>Ruby</li>
+            </ul>
+            <button class="seeproject-btn" type="button">See Project</button>
+          </div>
+          <div class="personal-projects">
+            <img src="images/image2.png" alt="project-image" />
+            <h1>Multi-Post Stories</h1>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul>
+              <li class="list-sumbua">
+                CSS <span> <img src="images/image3.png" alt="css" /></span>
+              </li>
+              <li class="list-sumbua">
+                Html <span> <img src="images/image3.png" alt="html" /></span>
+              </li>
+              <li class="list-sumbua">
+                Bootstrap
+                <span> <img src="images/image3.png" alt="bootstrap" /></span>
+              </li>
+              <li>Ruby</li>
+            </ul>
+            <button class="seeproject-btn" type="button">See Project</button>
+          </div>
+          <div class="personal-projects">
+            <img src="images/image2.png" alt="project-image" />
+            <h1>Multi-Post Stories</h1>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul>
+              <li class="list-sumbua">
+                CSS <span> <img src="images/image3.png" alt="css" /></span>
+              </li>
+              <li class="list-sumbua">
+                Html <span> <img src="images/image3.png" alt="html" /></span>
+              </li>
+              <li class="list-sumbua">
+                Bootstrap
+                <span> <img src="images/image3.png" alt="bootstrap" /></span>
+              </li>
+              <li>Ruby</li>
+            </ul>
+            <button class="seeproject-btn" type="button">See Project</button>
+          </div>
```

Comment:
- [ ] Kindly stick as much as possible to Figma design. Implement the `Epplise` on all the sections in your project.

## Figma design

![design](https://user-images.githubusercontent.com/81727988/164731618-4be30a4f-6d1d-4e47-9ae1-87b40a68752b.png)